changelog.md

```diff
@@ -13,6 +13,7 @@
 * Set macros on scope.macros & add new `macroexpand` helper to expand macro forms during
 compilation rather than uplifting macros to specials. (#258)
 * Add `macrodebug` util macro for printing expanded macro forms in REPL. (#258)
+* Make completion walk scope ancestors for macros & specials (#260)
```

Comment:
This is fine, but most of the time I like keeping the changelog to just user-visible changes, and it seems pretty unlikely that anyone would have noticed this one. =)

Reply:
Good point. Feel free to strip it out when you cut the release!